src/qgis_geonode/conf.py

```diff
@@ -251,5 +254,146 @@ def is_current_connection(self, connection_id: uuid.UUID):
     def _get_connection_settings_base(self, identifier: typing.Union[str, uuid.UUID]):
         return f"{self.BASE_GROUP_NAME}/connections/{str(identifier)}"
 
+    def store_current_search_filters(
+        self, current_filters: models.GeonodeApiSearchParameters
+    ):
+        with qgis_settings(
+            f"{self.BASE_GROUP_NAME}/{self.SEARCH_GROUP}/current_search_filters"
```

Comment:
IMO there is no need for another hierarchic level here.
```suggestion
            f"{self.BASE_GROUP_NAME}/current_search_filters"
```